titus-ext/aws/src/main/java/com/netflix/titus/ext/aws/AmazonAutoScalingAsyncProvider.java

```diff
@@ -33,7 +34,11 @@
 
     @Inject
     public AmazonAutoScalingAsyncProvider(AwsConfiguration configuration, AWSCredentialsProvider credentialProvider) {
-        String region = configuration.getRegion().trim().toLowerCase();
+        String region = configuration.getDataPlaneRegion().trim().toLowerCase();
```

Comment:
This code looks identical in all provider classes. We should move it to a utility class and share one implementation.